docs/pages/blog/material-ui-2024-updates.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Material UI 2024 end-of-year updates
+description: A look through Material UI during 2024, and a sneak peek of what's to come.
+date: 2024-12-11T00:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+As the end of the year approaches, we want to take some time to review some of this year's highlights for Material UI and update you on what the community can expect moving forward.
+
+## Material UI v6 release and support
+
+We released Material UI v6 in August, offering new features, fixes, and a smoother migration experience. So far, almost a million users have successfully migrated.
+
+The reception for the new version has been very positive, and we hope many more users can migrate in the upcoming months.
+
+Our focus has been on responding to v6-related issues to support your migration. We thank everyone in the community who has helped by reporting or fixing problems and encourage everyone else to migrate. We are happy and ready to help!
+
+## React 19 support
+
+As of today, Material UI v5 and v6 support React 19. This has been one of our main focuses during the second half of 2024.
+
+Next up, we'll update Material UI's dependencies and type versions once React 19 is stable.
+
+We hope you start trying it, and we're very grateful to the community members who have been and continue to test and report issues.
+
+## Material Design 3
+
+A year ago, we shared that we expected to have a Material Design 3 implementation by the end of 2024. Due to the efforts mentioned above, this has not been the case yet.
+
+Updating the look and feel in such a big way is very complex. We want to provide the best experience for everyone and we're currently discussing and working on the best way to achieve it. This will be one of our main focuses for 2025.
+
+## Looking towards 2025
+
+We're planning a new major release for the first half of 2025: Material UI v7.
+
+This version will fix issues that require breaking changes, including the long-awaited ESM support improvements, and other small fixes.
+
+Note that, while this major release won't contain visual updates since its only focus is fixing bugs, we'll keep exploring the best way to update the look and feel in parallel. We'll continue sharing any details as soon as we reach any decisions.
+
+## Follow along
+
+Feel free to leave your feedback on these new features and fixes in our [GitHub](https://github.com/mui/material-ui). We'd love to chat with you and are as excited as everyone to update how Material UI looks and feels.
```

Comment:
Is there a more specific call to action we could run with here? We don't really want to encourage folks to open issues to submit feedback unless it's something actionable on our end like a bug report or a request to improve the docs.

I would propose one CTA to either:

- direct users to upgrade to v6 if they haven't already, or
- ask for feedback on v7 / v8 plans (are there umbrella issues we could link to?)

Reply:
I'll add an invitation to upgrade, but we've decided not to open an issue for feedback for now

Reply:
Context on why there won't be a v7 issue: We need to keep the scope as tight as possible to focus on delivering v8 in time.